### PR TITLE
updates on the migration template

### PR DESCRIPTION
**Pull request by isacikgoz**
#### Summary
Some findings along the migration documentation has been applied:
- Drop/Create index requirements can be eliminated by adding them into the `pgLoader` configuration itself.
- We don't need to use `MATERIALIZE VIEWS` clause.
- Restore `SEARCH_PATH` after the migration, this was preventing mm-server to find `public` schema.

#### Release Note

```release-note
NONE
```

### CODE REVIEW

**Review by agnivade — APPROVED** [review]

File: server/tests/template.load [hunk 1]
```diff
@@ -26,12 +26,17 @@ CAST column Channels.Type to "channel_type" drop typemod,
...
 
-MATERIALIZE VIEWS exclude_products
-     excluding table names matching ~<IR_>, ~<focalboard>
+EXCLUDING TABLE NAMES MATCHING ~<IR_>, ~<focalboard>
 
 BEFORE LOAD DO
-     $$ ALTER SCHEMA public RENAME TO {{ .source_schema }}; $$
+     $$ ALTER SCHEMA public RENAME TO {{ .source_schema }}; $$,
+     $$ DROP INDEX IF EXISTS idx_posts_message_txt; $$,
+     $$ DROP INDEX IF EXISTS idx_fileinfo_content_txt; $$
```
Comment (streamer45) [inline server/tests/template.load:34]: What's the context behind these indexes?

**Review by streamer45 — APPROVED** [review]
Cool, left one question :+1:

File: server/tests/template.load (hunk 1 above)
Comment (isacikgoz) [reply server/tests/template.load:34]: @streamer45 These are actually related to: https://docs.mattermost.com/deploy/postgres-migration.html#full-text-indexes matter where `pgloader` will error and abort. We are just running this as a query later on and hopefully get it done.

Comment (streamer45) [reply server/tests/template.load:34]: Thanks :+1:

**Pull request merged**